### ISSUE TITLE
Generate new release to support 6.x Google Terraform provider

### ISSUE THREAD

**Issue by fancybear-dev** [terraform-google-modules/terraform-google-group#75]
### TL;DR

There is a commit in main, that needs to be released in order to support the new 6.x Google Terraform provider.

Merge this PR: https://github.com/terraform-google-modules/terraform-google-group/pull/74

### Expected behavior

A working module with 6.x.

Released 7 days ago: https://registry.terraform.io/providers/hashicorp/google/6.0.1

### Observed behavior

Due to version constrains, unable to initialize. 

### Terraform Configuration

```hcl
not relevant
```

### Terraform Version

```sh
not relevant
```

### Additional information

The commit is already pushed; https://github.com/terraform-google-modules/terraform-google-group/commit/65e8c6bcb754b3e849c4ce76101a43d8acdd1877

We just need a new release to be able to properly pull it.

**Comment by fancybear-dev** [comment]
closing as pr is merged 🫨🫨🫨 (thank you)
